string cam_id #ID of de camera to take the photo with

---------------
bool success #Success in the service execution
string message #Text indicating the separation result
